io.js install issues

It fails if I do npm install and if I try to do install mapnik@3.1.6 it fails saying the only valid install targers are <1.4.0

```
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.1.0","0.1.1","0.1.2","0.2.3","0.2.5","0.2.6","0.2.7","0.2.8","0.2.9","0.2.10","0.2.11","0.2.12","0.2.13","0.3.0","0.3.1","0.4.0","0.4.1","0.5.0","0.5.1","0.5.2","0.5.3","0.5.4","0.5.6","0.5.7","0.5.8","0.5.9","0.6.0","0.5.10","0.5.5","0.6.1","0.6.2","0.5.11","0.5.12","0.5.13","0.6.3","0.5.14","0.6.4","0.6.5","0.5.15","0.5.16","0.5.17","0.6.7","0.7.0","0.7.1","0.7.2","0.7.3","0.7.4","0.7.5","0.7.6","0.7.7","0.7.8","0.7.9","0.7.10","0.7.11","0.7.12","0.7.13","0.7.14","0.7.15","0.7.16","0.7.17","0.7.18","0.7.19","0.7.20","0.7.21","0.7.22","1.0.0","1.1.0","1.1.1","1.1.2","1.1.3","1.2.0","1.2.1","0.7.24","1.2.2","0.7.25","1.2.3","0.7.26","1.3.0-alpha","1.3.0-alpha2","1.3.0-alpha4","1.3.0","1.3.1","1.3.2","1.3.3","1.3.4","1.4.0-alpha","1.4.0","1.2.4","1.4.1","1.4.2","0.7.27","0.7.28","1.4.3","1.4.4","1.4.5","1.4.6","1.4.7","1.4.9","1.4.10"]
```

might this be related to the strict engine field in the package ?

--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
       "mapnik-shapeindex.js": "./bin/mapnik-shapeindex.js"
   },
   "engines": {
-      "node": ">= 0.8.0 < 0.11.0 || >= 0.11.13 < 0.13.0"
+      "node": ">= 0.8.0 < 0.11.0 || >= 0.11.13 < 0.13.0 || >= 1.2.0 < 2.0.0"
   },
   "engineStrict": true,
   "scripts"       : {

